Documentation/DocuBlocks/Rest/Cursors/post_api_cursor_identifier.md

```diff
@@ -77,13 +85,11 @@ Valid request for next batch
     };
     var response = logCurlRequest('POST', url, body);
 
-    var body = response.body.replace(/\\/g, '');
-    var _id = JSON.parse(body).id;
-    response = logCurlRequest('POST', url + '/' + _id, '');
+    response = logCurlRequest('POST', url + '/' + response.parsedBody.id, '');
     assert(response.code === 200);
 
     logJsonResponse(response);
-  ~ db._drop(cn);
+    db._drop(cn);
```

Comment:
Shouldn't the `~` remain here? I think it indicates that the output of the operation is not part of the generated example, i.e. here the `~` would hide the result of the `db._drop(cn);` operation, which is probably desired.

Reply:
Nope, that's only the case for `@EXAMPLE_ARANGOSH_OUTPUT` blocks but not `@EXAMPLE_ARANGOSH_RUN` blocks. Here, only the parts that are explicitly logged will be visible (`logCurlRequest()`, `logJsonResponse()`, etc.).